roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
We still have dependencies in the playbooks for most things master/node/etcd. It's probably safe to return those to their `meta/main.yml` files unless we want to provide a little speed boost for ansible-2.1.